scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
While running e2e tests, dependencies check takes place. Not defining suite here causes "Error: Unknown command line option".